src/test/jtx/TestHelpers.h

```diff
@@ -461,6 +461,101 @@ create(
 
 }  // namespace check
 
+static constexpr FeeLevel64 baseFeeLevel{256};
+static constexpr FeeLevel64 minEscalationFeeLevel = baseFeeLevel * 500;
+
+template <class Suite>
+void
+checkMetrics(
```

Comment:
I was recently introduced to https://en.cppreference.com/w/cpp/utility/source_location which should simplify the file and line sections of this helper function.

Reply:
Update: this might not work with minimum Clang versions (only supported with v14 🙁)